src/Microsoft.ML.TensorFlow/TensorflowCatalog.cs

```diff
@@ -45,23 +45,23 @@ public static class TensorflowCatalog
             => new TensorFlowEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnNames, inputColumnNames, tensorFlowModel);
 
         /// <summary>
-        /// Score or Retrain a tensorflow model (based on setting of the <see cref="TensorFlowTransformer.Arguments.ReTrain"/>) setting.
-        /// The model is specified in the <see cref="TensorFlowTransformer.Arguments.ModelLocation"/>.
+        /// Score or Retrain a tensorflow model (based on setting of the <see cref="TensorFlowTransformer.Options.ReTrain"/>) setting.
+        /// The model is specified in the <see cref="TensorFlowTransformer.Options.ModelLocation"/>.
```

Comment:
>Options [](start = 75, length = 7)

Please rerun locally RegenerateEntryPointCatalog and upload changes in core_ep-list.tsv to this PR.

Reply:
done

---
In reply to: [253190931](https://github.com/dotnet/machinelearning/pull/2312#discussion_r253190931) [](ancestors = 253190931)